### DISCUSSION TITLE
Removing (releasing) tags on the receiver side could be easier

### DISCUSSION THREAD

**Discussion by Debilski** [zrepl/zrepl#643] [Q&A]
I sometimes destroy old and stale file systems on the sender side, only to create them again with the same name. This means that future runs of zrepl won’t find a common snapshot and zrepl will abort with a planning error.
Assuming that I also don’t care about further backups of such a file system, I try to fix the issue by destroying them manually on the receiver side, only to run into the problem that there is a hold tag on one of the snapshots that I need to remove before.

Would it be possible to have zrepl help me untag these snapshots for me somehow? (I’d happily delete them myself for safety but finding the appropriate snapshots on which to remove the tags is somewhat tedious.)

**Comment by problame** [comment]
Check out the `zrepl zfs-abstraction --help` subcommand. Always run with `--help` first, since it's a pretty sharp knife.

**Comment by Debilski** [comment]
Ah, thanks. That looks quite helpful.
